src/PointwiseFunctions/AnalyticSolutions/Einstein/KerrSchild.cpp

```diff
@@ -21,7 +21,9 @@
 #include "Utilities/StdArrayHelpers.hpp"
 #include "Utilities/StdHelpers.hpp"
 
-namespace EinsteinSolutions {
+/// \cond
+namespace Einstein {
```

Comment:
Open question: do we have a `gr` namespace and would that be a better namespace nesting?

Reply:
we have one, so let's use it here too

Reply:
I'm fine leaving it, but also have no objection to changing it to `GeneralRelativity` like is done in `PointwiseFunctions`.

Reply:
I like `AnalyticFunctions/GeneralRelativity` better.